spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -228,10 +241,25 @@ defeats the purpose of SSL and makes you vulnerable to man-in-the-middle attacks
 ====
 
 [[configuration-security-basic-authentication]]
-=== Basic Authentication
+=== Traditional Authentication
+
+When using _traditional authentication_ Spring Cloud Data Flow will be the sole
+authentication provider. In that case Dataflow REST API users would use
+https://en.wikipedia.org/wiki/Basic_access_authentication[Basic Authentication]
+to access the endpoints.
+
+When using that option, users have a choice of 3 backing stores for authentication
+details:
 
-https://en.wikipedia.org/wiki/Basic_access_authentication[Basic Authentication] can
-be enabled by adding the following to `application.yml` or via
+* **Single User Authentication** using Spring Boot properties
+* **File-based authentication** for multiple users using a Yaml file
+* **Ldap Authentication**
+
+[[configuration-security-single-user-authentication]]
+==== Single User Authentication
+
+This is the simplest option and mimics the behavior of the default Spring Boot user.
+It can be enabled by adding the following to `application.yml` or via
```

Comment:
> <2> (Optional) The realm for Basic authentication. Will default to _Spring_ if not explicitly set.

Here, we are describing the realm as "Spring"; however, the actual string is "Spring Cloud Data Flow" - am I misinterpreting it?